include/alpaka/dev/DevUniformCudaHipRt.hpp

```diff
@@ -65,8 +65,16 @@ namespace alpaka
         {
             return !((*this) == rhs);
         }
+        int iDevice() const
+        {
+            return m_iDevice;
+        }
+        int setDevice(int devIdx) const
```

Comment:
I think `setDevice` should _not_ be `const`.

Reply:
Correct, updated the PR.